### PR TITLE
[HOLD] DEVELOPER-4049 Move all topics under /topics URL

### PR DESCRIPTION
**Pull request by adelasofia**
PLEASE REVIEW BUT DO NOT MERGE -- If approved it will be merged once the redirects are ready to go live

[DEVELOPER-4049 - Move all topics under '/topics' URL](https://issues.jboss.org/browse/DEVELOPER-4049)

- Updated main navigation Topics' links from '/<topic-name>' to '/topics/<topic-name>'
- Updated Acceptance tests
- Updated links in page--rhd-solution-overview.html.twig
- Updated DTM code

### CODE REVIEW

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/system/html.html.twig [hunk 1]
```diff
@@ -130,16 +130,12 @@
...
-          case 'web-and-api-development':
-          case 'dotnet':
-            pageType = 'topic';
-            contentType = primary;
+          case 'topic':
+            pageType = primary;
+            if(subs.length > 0) {
+              pageType = pageType.replace(/s$/,'');
+              contentType = subs[0];
+            }
```
Comment (adelasofia) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/system/html.html.twig]: @KamiQuasi do these DTM changes work for you?

**Comment by adelasofia** [comment]
retest this please

**Comment by LightGuard** [comment]
The code is fine. Are you planning on making all the changes to the individual nodes within Drupal?

**Comment by adelasofia** [comment]
@LightGuard yep I'll be making those changes in Drupal as soon as this is merged.

**Pull request merged**